test/sslapitest.c

```diff
@@ -1841,8 +1841,8 @@ static int test_cleanse_plaintext(void)
 #ifndef OPENSSL_NO_OCSP
 static int ocsp_server_cb(SSL *s, void *arg)
 {
+    STACK_OF(OCSP_RESPONSE) *sk_resp = NULL;
```

Comment:
Also this test should be generalized to cover the new feature: more then one OCSP response stapled.
I suggest testing with two of them, by adding a second one with a status different to `OCSP_RESPONSE_STATUS_SUCCESSFUL`.